packages/@aws-cdk/aws-chatbot/lib/slack-channel-configuration.ts

```diff
@@ -0,0 +1,283 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sns from '@aws-cdk/aws-sns';
+import * as cdk from '@aws-cdk/core';
+import { CfnSlackChannelConfiguration } from './chatbot.generated';
+
+/**
+ * Properties for a new Slack channel configuration
+ */
+export interface SlackChannelConfigurationProps {
```

Comment:
Much like Lambda functions, this resource emits CloudWatch metrics. It would be great to have a some `metric` methods e.g. https://github.com/aws/aws-cdk/blob/5a64bc518868301eb4c5ce8d2964d62d4a79a764/packages/%40aws-cdk/aws-lambda/lib/function-base.ts#L86 and the methods below it.
Please note, that unlike lambda, AWS Chatbot is a global resource that publishes metrics to `us-east-1` https://docs.aws.amazon.com/chatbot/latest/adminguide/monitoring-cloudwatch.html

Reply:
This resource also creates a log group, `/aws/chatbot/slackChannelConfigurationName`(if logging level is configured) when first invoked. It would be great to have an ability to modify that log group(put retention policy on it).
Same note about log groups being created in `us-east-1`

Reply:
Yeah, it can be extracted in separate PR.